### DISCUSSION TITLE
Standard Name Table Release - v86, Sep 2024

### DISCUSSION THREAD

**Discussion by efisher008** [cf-convention/discuss#353] [Announcements relevant to CF]
Dear All,

The standard name table has been updated. The current version is now version 86, dated 5 September 2024. The changes have been published on the NERC vocabulary server and a KWIC index for version 86 will be created in due course, so we ask for your patience in this. All changes are listed below.

Best wishes,
Ellie Fisher

**V86 standard name changes:**

**New names**

a. [#58](https://github.com/cf-convention/vocabularies/issues/58)
- `surface_upwelling_longwave_flux_in_air_assuming_clear_sky_and_reference_mole_fraction_of_ozone_in_air`
Proposed by Fiona O'Connor re. surface radiative flux diagnostic calls

b. [#52](https://github.com/cf-convention/vocabularies/issues/52)
- `modis_cloud_area_fraction`
- `modis_liquid_topped_cloud_area_fraction`
- `modis_ice_topped_cloud_area_fraction`
Proposed by Brandon Duran re. MODIS output from COSP simulator

c. [#49](https://github.com/cf-convention/vocabularies/issues/49), [#41](https://github.com/cf-convention/vocabularies/issues/41), [#38](https://github.com/cf-convention/vocabularies/issues/38), [#37](https://github.com/cf-convention/vocabularies/issues/37)
- `global_average_sea_level_change_due_to_change_in_ocean_mass`
- `change_in_sea_floor_height_above_reference_ellipsoid_due_to_ocean_tide_loading`
- `change_in_sea_surface_height_due_to_change_in_air_pressure`
- `ocean_dynamic_sea_level`
Proposed by Auore Biardeau re. sea surface height and sea level change

d. [#47](https://github.com/cf-convention/vocabularies/issues/47)
- `change_in_mean_sea_level_wrt_solid_surface`
Proposed by Swati Gehlot re. solid-Earth model VILMA / PalMod

e. [#46](https://github.com/cf-convention/vocabularies/issues/46)
- `product_of_wind_speed_and_water_vapor_saturation_deficit_in_air`
- `nfdrs_1_hour_fuel_moisture`
- `nfdrs_10_hour_fuel_moisture`
Proposed by Lee Kessenich re. wildland fire indexes

e. [#44](https://github.com/cf-convention/vocabularies/issues/44)
- `acoustic_equivalent_area_in_sea_water`
- `acoustic_inertia_in_sea_water`
- `acoustic_centre_of_mass_in_sea_water`
- `acoustic_index_of_aggregation_in_sea_water`
- `acoustic_proportion_occupied_in_sea_water`
Proposed by John Horne, Mark Yamane and Rob Cermak re. active acoustic information

f. [#16](https://github.com/cf-convention/vocabularies/issues/16)
- `mass_concentration_of_pm10_potassium_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_nitrate_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_sulfate_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_oxalate_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_bromide_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_methanesulfonic_acid_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_sodium_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_calcium_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_ammonium_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_chloride_dry_aerosol_particles_in_air`
- `mass_concentration_of_pm10_magnesium_dry_aerosol_particles_in_air`
Proposed by Jen Thomas re. dry aerosol particles in air

**Modified names**

a. [#21](https://github.com/cf-convention/vocabularies/issues/21) - updated term descriptions
- `received_power_of_radio_wave_in_air_scattered_by_air
`- `spectral_width_of_radio_wave_in_air_scattered_by_air`
- `radio_signal_roundtrip_travel_time_in_air`
Proposed by Ingemar Häggström re. atmospheric radar terms

b. [#8](https://github.com/cf-convention/vocabularies/issues/8)
- `upward_geothermal_heat_flux_at_ground_level_in_land_ice` -> `upward_heat_flux_at_base_of_grounded_ice_sheet`
Proposed by Martin Juckes re. heat fluxes out of bedrock.

**Comment by cofinoa** [comment]
@efisher008 and @japamment in the v86 the KWIC index it is missng. 

Please, can you provide the instructions on how to build the KWIC index? 

Moreover, can you describe the process how releases of standard name table versions are published and where?

Thank you
P.S: Probably there is a better "place" (discuss/issue) to discuss this.

**Comment by japamment** [comment]
Closing this to tidy up previous announcements.
